migrations/826-user-drop-source.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users DROP COLUMN `source`;
```

Comment:
```
MySQL [addons_mozilla_org]> select source, max(last_login_attempt) from users group by source order by source;^A
+--------+-------------------------+
| source | max(last_login_attempt) |
+--------+-------------------------+
|      0 | 2015-08-19 13:37:11     |
|      1 | 2015-08-19 06:27:05     |
|      2 | 2015-08-19 13:10:36     |
+--------+-------------------------+
```

Maybe we should just keep the column for now and decide how to deal with it later.